src/mod.ts

```diff
@@ -0,0 +1,128 @@
+export type {
+  Manifest,
+  ManifestWasm,
+  ManifestWasmUrl,
+  ManifestWasmData,
+  ManifestLike as IntoManifest
+} from './manifest.ts';
+import { FEATURES } from 'js-sdk:features';
+export { FEATURES } from 'js-sdk:features';
+
+import {
+  ManifestLike,
+  intoManifest as _intoManifest,
+  toWasmModuleData as _toWasmModuleData
+} from './manifest.ts';
+
+import { createForegroundPlugin as _createForegroundPlugin } from './foreground-plugin.ts';
+import { createBackgroundPlugin as _createBackgroundPlugin } from './background-plugin.ts';
+export { type CallContext } from './call-context.ts';
+
+/**
+ * {@link ExtismPlugin} Config
+ */
+export interface PluginConfigLike {
+  [key: string]: string
+}
+export type PluginConfig = { [key: string]: string };
+
+/**
+ * Options for initializing an Extism plugin.
+ */
+export interface ExtismPluginOptions {
+  useWasi?: boolean | undefined;
+  offMainThread?: boolean | undefined;
```

Comment:
This variable name makes sense, but when said verbally, I feel like saying something runs "off of the main thread" could be interpreted as running "on the main thread" :laughing:

Maybe `runInWorker` ? Also fine with keeping as is

Reply:
Ooh, yes, great suggestion! I tripped over `offMainThread` a lot too.